dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/HealthCenterTest.java

```diff
@@ -53,7 +53,7 @@ public static void tearDown() throws Exception {
     @Test
     public void testHealthCenterInfo() throws Exception {
         Assert.assertFalse("Expected healthcenter INFO message",
-                           server.findStringsInLogs("^INFO:.*com\\.ibm\\.java\\.diagnostics\\.healthcenter\\.agent\\.iiop\\.port",
+                           server.findStringsInLogs("INFO:.*Health Center agent started on port*",
```

Comment:
Do you need the trailing `*` or did you mean to put `.*`?

Reply:
No, we don't need the trailing `*`, since it should match the first string, the rest could be any port number.
Updated the PR to remove the trailing `*`.